SchedulerJobs/SchedulerJobs.Common/Logging/LoggerAdapter.cs

```diff
@@ -0,0 +1,311 @@
+namespace SchedulerJobs.Common.Logging
+{
+    using System;
+    using System.Collections.Generic;
+    using Microsoft.Extensions.Logging;
+
+    public static partial class LoggerAdapter
```

Comment:
this class is quite large - any chance it could be split into multiple classes, perhaps one per namespace?

eg:

JobLogger
ServiceLogger

if you stick with partial classes, you won't need to update the calls

Reply:
done